static/css/components/iaBar.less

```diff
@@ -7,8 +7,13 @@
   max-width: 960px;
   margin: 0 auto;
   -webkit-box-align: center;
-  -ms-flex-align: center;
+  -ms-flex-align: center;  
   align-items: center;
+  &Donate{
+    align-items: baseline;
```

Comment:
This is not affecting the layout, as the `iaBarDonate` element is not a flex or grid container.